src/requirements.txt

```diff
@@ -1,26 +1,26 @@
 -i https://pypi.python.org/simple
-asgiref==3.6.0 ; python_version >= '3.7'
-boto3==1.26.69
-botocore==1.29.69 ; python_version >= '3.7'
-cachetools==5.3.0
-certifi==2022.12.7 ; python_version >= '3.6'
+asgiref==3.7.2 ; python_version >= '3.7'
+boto3==1.26.144
+botocore==1.29.144 ; python_version >= '3.7'
+cachetools==5.3.1
+certifi==2023.5.7 ; python_version >= '3.6'
 cfenv==0.5.3
 cffi==1.15.1
-charset-normalizer==3.0.1 ; python_version >= '3.6'
-cryptography==39.0.1 ; python_version >= '3.6'
+charset-normalizer==3.1.0 ; python_full_version >= '3.7.0'
+cryptography==41.0.1 ; python_version >= '3.7'
```

Comment:
🧵 Major version bump

Reply:
@neilmb noted on the previous PR: https://github.com/cisagov/getgov/pull/667#issuecomment-1572558324

Reply:
From https://cryptography.io/en/latest/changelog/

Since our previous version (39.0.1), the following breaking changes are listed:

> - BACKWARDS INCOMPATIBLE: Support for OpenSSL less than 1.1.1d has been removed. Users on older version of OpenSSL will need to upgrade.
> - BACKWARDS INCOMPATIBLE: Support for Python 3.6 has been removed.
> - BACKWARDS INCOMPATIBLE: Dropped support for LibreSSL < 3.6.
> - BACKWARDS INCOMPATIBLE: As announced in the 39.0.0 changelog, the way cryptography links OpenSSL has changed. This only impacts users who build cryptography from source (i.e., not from a wheel), and specify their own version of OpenSSL. For those users, the CFLAGS, LDFLAGS, INCLUDE, LIB, and CRYPTOGRAPHY_SUPPRESS_LINK_FLAGS environment variables are no longer valid. Instead, users need to configure their builds [as documented here](https://docs.rs/openssl/latest/openssl/#automatic).